### PR TITLE
[CPLP-863]- #3 - membership flag implementation

### PR DESCRIPTION
**Pull request by smartdeepp**

### CODE REVIEW

File: cx-portal/public/testdata/partnerNetwork/membershipData.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+  "BPNL00000000D5OM",
+  "BPNL00000000CYTH",
+  "BPNL00000000CRYC",
+  "BPNL00000000DEID",
+  "BPNL00000000B540"
+]
```
Comment (jjeroch) [inline cx-portal/public/testdata/partnerNetwork/membershipData.json:7]: @smartdeepp whats that for?

File: cx-portal/public/testdata/partnerNetwork/businessPartnersDataGrid.json [hunk 2]
```diff
@@ -69,6 +70,7 @@
     "street": "R RIO DE JANEIRO , 421",
     "zipCode": "13800-470",
     "city": "Moji Mirim",
+    "cxmember": true,
     "identifiers": [
       {
         "uuid": "d38cc9b0-e9c8-4a65-a61f-f65fdc89c0d0",
```
Comment (jjeroch) [inline cx-portal/public/testdata/partnerNetwork/businessPartnersDataGrid.json:76]: @smartdeepp can you please check if we still need those test data? We dont use them anymore, or are we fetching them for unit tests?

Comment (smartdeepp) [reply cx-portal/public/testdata/partnerNetwork/businessPartnersDataGrid.json:76]: Yes, @jjeroch  , we are using dummy data for unit tests; the same applies to the comment above.

In addition, one of the github actions failed the unit test, so I had to make the necessary modifications.

**Comment by jjeroch** [comment]
@lavanya-bmw @manojava-gk please review
@smartdeepp there are conflicts. can you please cleanup the conflicts

File: cx-portal/public/testdata/partnerNetwork/businessPartnersDataGrid.json (hunk 2 above)
Comment (jjeroch) [reply cx-portal/public/testdata/partnerNetwork/businessPartnersDataGrid.json:76]: fine

**Comment by smartdeepp** [comment]
Hello, @lavanya-bmw @manojava-gk, conflicts resolved as per the comment.

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
please add your changes to the RELEASENOTES.md file with every PR so we have an overview by the end of the sprint

**Comment by smartdeepp** [comment]
> please add your changes to the RELEASENOTES.md file with every PR so we have an overview by the end of the sprint

Updated.

**Review by lavanya-bmw — APPROVED** [review]
looks good to me

**Comment by jjeroch** [comment]
@thackerronak please fix the outstanding release note conflicts

**Comment by smartdeepp** [comment]
> @thackerronak please fix the outstanding release note conflicts

updated.

**Review by lavanya-bmw — APPROVED** [review]

**Review by jjeroch — APPROVED** [review]

**Pull request merged**